tasks_week3_Shai_Axelrod/Zoo/include/Animal.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+#include "Location.h"
+#include "Direction.h"
+/**
+ * @class Animal
+ * @brief Represents an animal in the simulation.
+ */
+class Animal {
+protected:
+    int steps; /**< The number of steps the takes in each move. */
+    std::string name; /**< The name of the animal. */
```

Comment:
members should start with underscore ' _ '

Reply:
AFAIK its not an agreed convention in cpp and there are those who oppose it

Reply:
you are the use of underscores as a prefix for class members is not a universally adopted convention but it's a coding style followed by a lot of developers and organizations, and so are we. 
The purpose of using an underscore prefix is typically to distinguish class member variables from local variables or parameters within member functions.
check with barak